Unit-11/main.js

```diff
@@ -0,0 +1,97 @@
+// exercise-1
+const createElementMethodOne = (char,number) => {
+  const newArray = []
+  for(let i = 0;  i < number; i++) {
+    newArray.push(char)
+  }
+  return newArray
+}
+// exercise-1
+const createElementMethodTwo = (char,number) => {
+  const newArray = []
+  let i=0
+  while(i < number) {
+    newArray.push(char)
+    i+=1
+  }
+  return newArray
+}
+
+// exercise-2
+const createArrayReverse = (array = []) => {
+  const reverseArray = []
+  for(let i=array.length-1; i >= 0; i--) {
+    reverseArray.push(array[i])
+  }
+  return reverseArray
+}
+// exercise-3
+const deleteArray = (array = []) => {
+  const trueValues = []
+  for(let i = 0; i < array.length; i++) {
+    if(typeof(array[i]) == 'number' && array[i] !=0) {
+      trueValues.push(array[i])
+    }
+  }
+  return trueValues
+}
+
+// exercise-4
+const createObject = (array = []) => {
+  const newObject = {}
+  array.forEach(item => {
+    newObject[item[0]] = item[1]
+  })
+  return newObject
+}
+// exercise-5
+const sortArraysAscending = (array = []) => {
+  for(let i = 0; i < array.length-1; i++) {
+    for(let j = i+1; j < array.length; j++) {
+      if(array[i] > array[j]) {
+        let temp = array[i]
+        array[i] = array[j]
+        array[j] = temp
+      }
+    }
+  }
+  return array
+}
+// em hỏi là tại sao ở vòng for thứ 2 em thay j=1 thì code của em nó chạy sai ạ?
+
+// exercise-6
+const checkObject = a => {
+  if (typeof(a) !== 'object') {
+    return false
+  }
+  if (Array.isArray(a) == true) {
+    return false
+  } else {
+    return true
+  }
+}
+
+// exercise-8
+const deleteItemInArray = (array = []) => {
+  if(array.length < 5) {
+    return 'Array phải có nhiều hơn 5 phần tử'
+  }
+  else {
```

Comment:
Cách viết như thế này
```suggestion
  } else {
```